newEvaluation.html

```diff
@@ -0,0 +1,394 @@
+						</select>
+					</li>
+					<li>
+						<label>Object manipulation</label>
+						<select name = "objectManipulation">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>DOM Manipulation</label>
+						<select name = "domManipulation">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Functions</label>
+						<select name = "functions">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Templating</label>
+						<select name = "templating">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Event Handling</label>
+						<select name = "eventHandling">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Prototype & OOP</label>
+						<select name = "prototypeoop">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Testing (unit, E2E)</label>
+						<select name = "testing">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>AJAX</label>
+						<select name = "ajax">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Debugging</label>
+						<select name = "debugging">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Websockets</label>
+						<select name = "websockets">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Tooling</label>
+						<select name = "tooling">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Libraries</label>
+						<select name = "libraries">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Promises</label>
+						<select name = "promises">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Browser Engines</label>
+						<select name = "browserEngines">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Frameworks</label>
+						<select name = "frameworks">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+				</ul>
+			</fieldset>
+			
+			<fieldset>
+				<legend>NodeJS</legend>
+				<ul>
+					<li>
+						<label>Backend frameworks</label>
+						<select name = "backendFrameworks">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Templating</label>
+						<select name = "templating">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>DOM Manipulation</label>
+						<select name = "domManipulation">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Unit Testing</label>
```

Comment:
Whoa! Do you guys realize how much stuff we still have to teach you? :open_mouth: